cn0534/build_arm.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+ROOTFS=$(pwd)/pluto-0.30.sysroot
+CC_DIR=/home/guest/linux_dev/gcc-arm-8.3-2019.03-x86_64-arm-linux-gnueabihf/bin
```

Comment:
maybe automate this with something like 
https://github.com/analogdevicesinc/wiki-scripts/blob/master/linux/build_rpi_kernel_image.sh#L18-L39

Reply:
Edited the build script using the link as given as reference. Updated through 7969679.